.github/workflows/build.yml

```diff
@@ -17,12 +17,40 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          repository: llvm/llvm-project
+          ref: llvmorg-13.0.0-rc1
+          path: llvm-project
+          fetch-depth: 1
+      - name: Get LLVM Revision
+        id: llvm-revision
+        run: |
+          echo "::set-output name=revision::$(git -C ${{ github.workspace }}/llvm-project rev-parse HEAD)"
+      - uses: actions/cache@v2
+        id: llvm-build
+        with:
+          path: ${{ github.workspace }}/build/llvm-project
+          key: ${{ runner.os }}-${{ matrix.arch }}-${{ steps.llvm-revision.outputs.revision }}
+      - name: configure LLVM
+        if: steps.llvm-build.outputs.cache-hit != 'true'
+        run: cmake -B ${{ github.workspace }}/build/llvm-project -D CMAKE_BUILD_TYPE=Release -G "Visual Studio 16 2019" -S ${{ github.workspace }}/llvm-project/llvm
+      - name: build LLVM tools
+        if: steps.llvm-build.outputs.cache-hit != 'true'
+        run: |
+          cmake --build ${{ github.workspace }}/build/llvm-project --config Release --target FileCheck
+          cmake --build ${{ github.workspace }}/build/llvm-project --config Release --target yaml2obj
```

Comment:
Would it be an idea to also create artifacts from this, so contributors can download those and configure it locally?
When going that route, I would suggest to also include `obj2yaml` so that new tests can be added somewhat simpler.

Reply:
Or even to create a new build definition,
only triggered manually that builds the llvm binaries, and attached it to a 'Tools' release in this repo.
Then this tools release can be checked out in the builder, so that it does not have to rely on the github cache, and humans can easily grab the used tooling as well.
This will also ensure that the used tooling does not change overnight when there is a cache miss.

Reply:
(and bring build times down by ~10 minutes for cache misses)